rules/opentitan.bzl

```diff
@@ -389,41 +640,55 @@ def _unique_deps(*deplists):
 
 def opentitan_functest(
         name,
-        platform = OPENTITAN_PLATFORM,
         targets = ["verilator", "cw310"],
         args = [],
         data = [],
         ottf = _OTTF_DEPS,
+        test_in_rom = False,
+        signed = False,
+        key = "test_key_0",
         verilator = None,
         cw310 = None,
         **kwargs):
     """A helper macro for generating OpenTitan functional tests.
-    This macro is mostly a wrapper around opentitan_binary, but creates
-    testing artifacts for each of the keys in `per_device_deps`.
-    The testing artifacts are then given to an `sh_test` rule which
-    dispatches the test via opentitantool.
+
+    This macro is mostly a wrapper around opentitan_flash_binary, but creates
+    testing artifacts for each of the hardware targets in `targets`. The testing
+    artifacts are then given to an `sh_test` rule which dispatches the test to
+    the corresponding hardware target via opentitantool.
     Args:
       @param name: The name of this rule.
-      @param platform: The target platform for the artifacts.
-      @param targets: A list of targets on which to dispatch tests.
+      @param targets: A list of hardware targets on which to dispatch tests.
```

Comment:
Thanks, it looks like the term targets has gotten overloaded and this clarifies it some!